config.yml

```diff
@@ -5,7 +5,6 @@ Defaults:
     authorizers:
```

Comment:
Is https://github.com/djmitche/taskcluster-login/blob/ab01f586c6384a1f67db90c8caa7d636817a002b/config.yml#L3 still needed:

```
authenticators:
  - auth0
```